scripts/check_cli_bench_regression.sh

```
@@ -0,0 +1,221 @@
+        if [[ "${i}" -eq "${attempts}" ]]; then
+            echo "::error::benchstat install failed after ${attempts} attempts"
+            return 1
+        fi
+        echo "::warning::benchstat install attempt ${i} failed; retrying in ${delay}s..."
+        sleep "${delay}"
+        delay=$((delay + 5))
+    done
+    # GOPATH/bin is on PATH via setup-go's default PATH config in
+    # GitHub Actions; on a local shell it must already be present.
+    if ! command -v benchstat >/dev/null 2>&1; then
+        export PATH="${PATH}:$(go env GOPATH)/bin"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)